app/src/main/java/com/pvp/app/service/NotificationReceiver.kt

```diff
@@ -0,0 +1,76 @@
+package com.pvp.app.service
+
+import android.Manifest
+import android.app.AlarmManager
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.os.Build
+import androidx.core.app.ActivityCompat
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.pvp.app.R
+
+class NotificationReceiver : BroadcastReceiver() {
+    override fun onReceive(context: Context, intent: Intent) {
+        showNotification(
+            context,
+            intent.getStringExtra("notificationText"),
+            intent.getIntExtra("notificationId", 0)
+        )
+    }
+
+    private fun showNotification(context: Context, text: String?, notificationId: Int) {
+        val notificationManager = NotificationManagerCompat.from(context)
+        val channelId = "Task"
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            val channel = NotificationChannel(
+                channelId,
+                "Task",
+                NotificationManager.IMPORTANCE_DEFAULT
+            )
+            notificationManager.createNotificationChannel(channel)
+        }
+
+        val notification = NotificationCompat.Builder(context, channelId)
+            .setContentTitle("Calendar reminder")
+            .setContentText(text)
+            .setSmallIcon(R.drawable.ic_launcher_foreground)
+            .setPriority(NotificationCompat.PRIORITY_DEFAULT)
+            .build()
+
+        if (ActivityCompat.checkSelfPermission(
+                context,
+                Manifest.permission.POST_NOTIFICATIONS
+            ) != PackageManager.PERMISSION_GRANTED
+        ) {
+            return
+        }
+
+        notificationManager.notify(notificationId, notification)
+    }
+}
+
+fun scheduleNotification(context: Context, text: String, delaySeconds: Int, notificationId: Int) {
+    println("scheduled after ${delaySeconds}, notification_id = $notificationId")
```

Comment:
Prints used for debugging should not be kept. But if it is, then separate variables from method call by a single empty line.